packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -75,27 +87,24 @@ import VerificationSuccessScreen from 'src/verify/VerificationSuccessScreen'
 
 const Stack = createStackNavigator<StackParamList>()
 
-export const headerArea = {
+export const defaultScreenOptions = {
   // Force this for now on iOS so screen transitions look normal
   // given we intentionally hide the bottom separator from the nav bar
```

Comment:
Looks like this comment can be removed since it was related to the `headerMode` property.